### PR TITLE
Move contents of app to gps/app

### PR DESCRIPTION
**Pull request by mattThousand**
#### Description:

- Moves contents of `app` to `gps/app`
- Updates snapshots

### CODE REVIEW

**Comment by tstirrat** [comment]
`yarn run-ios` yields:

```
builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -resolve-src-symlinks /Users/tstirrat/dev/safe-paths/shared/assets/fonts/IBMPlexSans.ttf /Users/tstirrat/Library/Developer/Xcode/DerivedData/COVIDSafePaths-cgplwspxdnrjsicwjhwxqabcxsup/Build/Products/Debug-iphonesimulator/COVIDSafePaths.app
error: /Users/tstirrat/dev/safe-paths/shared/assets/fonts/IBMPlexSans.ttf: No such file or directory (in target 'COVIDSafePaths' from project 'COVIDSafePaths')


** BUILD FAILED **
```

**Comment by tstirrat** [comment]
I also tried with a super clean repo:

```
$ git clean -f -d -X

yarn run-ios

... snip

Installing fastlane 2.146.1
Bundle complete! 4 Gemfile dependencies, 99 gems now installed.
Bundled gems are installed into `./vendor/bundle`

[!] Invalid `Podfile` file: cannot load such file -- /Users/tstirrat/dev/safe-paths/node_modules/@react-native-community/cli-platform-ios/native_modules.

 #  from /Users/tstirrat/dev/safe-paths/ios/Podfile:2
 #  -------------------------------------------
 #  platform :ios, '9.0'
 >  require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 #
 #  -------------------------------------------
```

**Comment by mattThousand** [comment]
> I also tried with a super clean repo:
> 
> ```
> $ git clean -f -d -X
> 
> yarn run-ios
> 
> ... snip
> 
> Installing fastlane 2.146.1
> Bundle complete! 4 Gemfile dependencies, 99 gems now installed.
> Bundled gems are installed into `./vendor/bundle`
> 
> [!] Invalid `Podfile` file: cannot load such file -- /Users/tstirrat/dev/safe-paths/node_modules/@react-native-community/cli-platform-ios/native_modules.
> 
>  #  from /Users/tstirrat/dev/safe-paths/ios/Podfile:2
>  #  -------------------------------------------
>  #  platform :ios, '9.0'
>  >  require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
>  #
>  #  -------------------------------------------
> ```

👀 🧑‍💻 🕙

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "run-ios": "yarn install:pod && react-native run-ios",
     "preinstall": "node -e \"if(process.env.npm_execpath.indexOf('yarn') === -1) throw new Error('You must use Yarn to install, not NPM')\"",
     "install:pod": "cd ios && bundle install && bundle exec pod install",
+    "install:all": "find . -type dir -name node_modules | xargs rm -rf && yarn",
```
Comment (tstirrat) [inline package.json]: deleting the `node_modules/` will invalidate the caches which speed up the workflows... ideally we can avoid this.

Comment (mattThousand) [reply package.json]: `find . -type dir -name node_modules | yarn` 🤔

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -100,7 +100,7 @@ jobs:
           bundle install --jobs 4 --retry 3
         working-directory: ./ios
 
-      - run: pod install --repo-update
```
Comment (tstirrat) [inline .github/workflows/test.yml]: undo this change. was it necessary?

File: shared/assets/svgs/backArrow.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+export default `
```
Comment (tstirrat) [inline shared/assets/svgs/backArrow.js]: I don;t think these are actually being used. because the consumer js files are using relative imports

File: .github/workflows/test.yml (hunk 2 above)
Comment (mattThousand) [reply .github/workflows/test.yml]: hm, are you sure we want to pass `repo-update`? It will pull down the latest available version if the Podfile.lock does not specify otherwise

File: i18next-parser.config.js [hunk 4]
```diff
@@ -45,7 +45,7 @@ module.exports = {
   // Namespace separator used in your translation keys
   // If you want to use plain english keys, separators such as `.` and `:` will conflict. You might want to set `keySeparator: false` and `namespaceSeparator: false`. That way, `t('Status: Loading...')` will not think that there are a namespace and three separator dots for instance.
 
-  output: 'app/locales/$LOCALE.json',
+  output: 'gps/app/locales/$LOCALE.json',
```
Comment (tstirrat) [inline i18next-parser.config.js:48]: Also the `input:` key below
